--- src/windows/Installer.Windows/layout.ps1
+++ src/windows/Installer.Windows/layout.ps1
@@ -44,35 +44,29 @@
 	--framework net472 `
 	--configuration "$CONFIGURATION" `
 	--runtime win-x86 `
-	--output "$PAYLOAD"
+	--output "$PAYLOAD" `
+	-p:UseAppHost=false
 
 Write-Output "Publishing core UI helper..."
 dotnet publish "$GCM_UI_SRC" `
 	--configuration "$CONFIGURATION" `
 	--output "$PAYLOAD"
 
 Write-Output "Publishing Bitbucket UI helper..."
 dotnet publish "$BITBUCKET_UI_SRC" `
 	--configuration "$CONFIGURATION" `
 	--output "$PAYLOAD" 
 
 Write-Output "Publishing GitHub UI helper..."
 dotnet publish "$GITHUB_UI_SRC" `
 	--configuration "$CONFIGURATION" `
 	--output "$PAYLOAD" 
 
 Write-Output "Publishing GitLab UI helper..."
 dotnet publish "$GITLAB_UI_SRC" `
 	--configuration "$CONFIGURATION" `
 	--output "$PAYLOAD" 
 
-# Create copy of main GCM executable with older "GCM Core" name
-Copy-Item -Path "$PAYLOAD/git-credential-manager.exe" `
-	-Destination "$PAYLOAD/git-credential-manager-core.exe"
-
-Copy-Item -Path "$PAYLOAD/git-credential-manager.exe.config" `
-	-Destination "$PAYLOAD/git-credential-manager-core.exe.config"
-
 # Delete libraries that are not needed for Windows but find their way
 # into the publish output.
 Remove-Item -Path "$PAYLOAD/*.dylib" -Force
