--- src/linux/Packaging.Linux/layout.sh
+++ src/linux/Packaging.Linux/layout.sh
@@ -95,3 +95,3 @@
         --runtime="$RUNTIME" \
-        --self-contained \
+        --no-self-contained \
         -p:PublishSingleFile=true \
--- src/osx/Installer.Mac/layout.sh
+++ src/osx/Installer.Mac/layout.sh
@@ -99,3 +99,3 @@
	--runtime="$RUNTIME" \
-	--self-contained \
+	--no-self-contained \
	--output="$(make_absolute "$PAYLOAD")" || exit 1
--- src/windows/Installer.Windows/layout.ps1
+++ src/windows/Installer.Windows/layout.ps1
@@ -62,3 +62,4 @@
 dotnet publish "$GCM_SRC" `
+	--no-self-contained `
 	--framework net472 `
 	--configuration "$Configuration" `
