--- src/windows/Core.UI.Windows/Core.UI.Windows.csproj
+++ src/windows/Core.UI.Windows/Core.UI.Windows.csproj
@@ -4,1 +4,1 @@
-    <TargetFramework>net472</TargetFramework>
+    <TargetFramework>net472-windows</TargetFramework>
--- src/windows/Atlassian.Bitbucket.UI.Windows/Atlassian.Bitbucket.UI.Windows.csproj
+++ src/windows/Atlassian.Bitbucket.UI.Windows/Atlassian.Bitbucket.UI.Windows.csproj
@@ -5,1 +5,1 @@
-    <TargetFramework>net472</TargetFramework>
+    <TargetFramework>net472-windows</TargetFramework>
--- src/windows/GitHub.UI.Windows/GitHub.UI.Windows.csproj
+++ src/windows/GitHub.UI.Windows/GitHub.UI.Windows.csproj
@@ -5,1 +5,1 @@
-    <TargetFramework>net472</TargetFramework>
+    <TargetFramework>net472-windows</TargetFramework>
--- src/windows/GitLab.UI.Windows/GitLab.UI.Windows.csproj
+++ src/windows/GitLab.UI.Windows/GitLab.UI.Windows.csproj
@@ -5,1 +5,1 @@
-    <TargetFramework>net472</TargetFramework>
+    <TargetFramework>net472-windows</TargetFramework>
--- src/windows/Git-Credential-Manager.UI.Windows/Git-Credential-Manager.UI.Windows.csproj
+++ src/windows/Git-Credential-Manager.UI.Windows/Git-Credential-Manager.UI.Windows.csproj
@@ -5,1 +5,1 @@
-    <TargetFramework>net472</TargetFramework>
+    <TargetFramework>net472-windows</TargetFramework>
